contrib/checkbox-provider-ce-oem/series_uc22/snap/snapcraft.yaml

```
@@ -0,0 +1,136 @@
+name: checkbox-ce-oem
+summary: Checkbox CE OEM test runner and public providers
+description: "Checkbox CE OEM test runner and public providers"
+confinement: strict
+grade: stable
+
+version: '0.12-jammy'
```

How about starting the version from 1.0?
```suggestion
version: '1.0-jammy'
```